meta-ivi/recipes-support-ivi/keyutils/keyutils_1.5.9.bb

```diff
@@ -16,6 +16,7 @@ SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git;p
           file://keyutils-fix-the-cflags-for-all-of-targets.patch \
           "
 SRC_URI_append_arm = " file://keyutils-arm-remove-m32-m64.patch"
+SRC_URI_append_aarch64 = " file://keyutils-arm-remove-m32-m64.patch"
 SRC_URI_append_x86 = " file://keyutils_fix_x86_cflags.patch"
 SRC_URI_append_x86-64 = " file://keyutils_fix_x86-64_cflags.patch"
 SRC_URI_append_powerpc = "file://keyutils-fix-powerpc-cflags.patch"
```

Comment:
Hi Changhyeok

move all machine dependent code to meta-ivi-bsp layer.

thank you.

__
Yong-iL Joh, Sr.MTS, Wind River
mobile +82-10-6332-6692  skype tolkienj

From: Changhyeok Bae [mailto:notifications@github.com]
Sent: Tuesday, June 21, 2016 1:58 AM
To: GENIVI/meta-ivi
Cc: Joh, Yong-Il (Tolkien); Mention
Subject: Re: [GENIVI/meta-ivi] keyutils: Fix build error in arm 64bit (#4)


In meta-ivi/recipes-support-ivi/keyutils/keyutils_1.5.9.bb<https://github.com/GENIVI/meta-ivi/pull/4#discussion_r67725510>:

> @@ -16,6 +16,7 @@ SRC_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git;p

>            file://keyutils-fix-the-cflags-for-all-of-targets.patch<file:///\\keyutils-fix-the-cflags-for-all-of-targets.patch> \

>            "

>  SRC_URI_append_arm = " file://keyutils-arm-remove-m32-m64.patch<file:///\\keyutils-arm-remove-m32-m64.patch>"

> +SRC_URI_append_aarch64 = " file://keyutils-arm-remove-m32-m64.patch<file:///\\keyutils-arm-remove-m32-m64.patch>"

>  SRC_URI_append_x86 = " file://keyutils_fix_x86_cflags.patch<file:///\\keyutils_fix_x86_cflags.patch>"

>  SRC_URI_append_x86-64 = " file://keyutils_fix_x86-64_cflags.patch<file:///\\keyutils_fix_x86-64_cflags.patch>"

>  SRC_URI_append_powerpc = "file://keyutils-fix-powerpc-cflags.patch<file:///\\keyutils-fix-powerpc-cflags.patch>"

@tolkien-joh<https://github.com/tolkien-joh>
Another SRC_URI_append has machine dependency. So should I move another SRC_URI_appends? or only my patch?

Thanks

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/GENIVI/meta-ivi/pull/4/files/36021251548c39e8cba0e25d9104d6ab117b5441#r67725510>, or mute the thread<https://github.com/notifications/unsubscribe/ARr8X-2_KrgHSeFENeN3aiKCiho4SIu7ks5qNsaLgaJpZM4I43k5>.